runtime/onert/core/src/ir/train/UseDefGenerator.cc

```diff
@@ -103,6 +103,27 @@ void UseDefGenerator::visit(const train::operation::Loss &node)
   usedef_chain.removeTrainingUse(backwarding_op_index);
 }
 
+void UseDefGenerator::visit(const train::operation::Reshape &node)
+{
+  assert(_node_to_idx.find(&node) != _node_to_idx.end());
+  const auto &op_index = _node_to_idx.at(&node);
+  const auto backwarding_op_index = TrainingOperationIndex{op_index, false};
+
+  // Insert uses of forwarding output
+  const auto &out_index = node.getOutputs().at(0);
+  const auto out_forwarding_index = TrainingOperandIndex{out_index, true};
+  insertUse(out_forwarding_index, backwarding_op_index);
```

Comment:
Does `reshape` use `out_forwarding_index`?

Reply:
I checked `train::ReshapeLayer`, but it does not access output for forwarding. :thinking: 

https://github.com/Samsung/ONE/blob/44fd15bf5c6df7c13a8abbea01b1d4d6a4d5ff89/runtime/onert/backend/train/ops/ReshapeLayer.cc#L59

Reply:
It's my mistake I will fix it.